src/main/frontend/handler/editor.cljs

```diff
@@ -2906,15 +2918,6 @@
            (not (string/blank? (util/get-selected-text))))
       (html-link-format! text)
 
-      (and (gp-util/url? text)
-           (boolean (text/get-matched-video text)))
-      (commands/simple-insert! (state/get-edit-input-id) (util/format "{{video %s}}" text) nil)
-      
-      (and (gp-util/url? text)
```

Comment:
Looks like this one breaks the wrapping of pasted YT links on mobile (and cmd+shift+v not really available there)